ACMILabs/interactive-label
# #124 Add notification bar

Resolves #124

We want to encourage visitors to tap an image before tapping their lens.

### Acceptance Criteria
- [x] Add a notification bar to interactive labels to prompt visitors to tap images before tapping their lens

### Relevant design files
* None

### Testing instructions
1. **Note**: There weren't any tests previously, and testing thoroughly is very difficult with the current arrangement of the `app.js` file
1. Point your interactive label to XOS Staging (see below)
1. Start your environment: `cd development` and `docker-compose up --build`
1. Edit `app.js` to set `notification_bar_timeout = 3000` (3 seconds to make it easier to test)
1. Note the notification bar appears in ~3 seconds: http://localhost:8081
1. Test this on a Staging device in: [s__interactive-label-x86](https://dashboard.balena-cloud.com/fleets/1765718/devices)

```env
XOS_API_ENDPOINT=https://xos-stg.acmi.net.au/api/
AUTH_TOKEN=<SNIP>
# FSF
XOS_PLAYLIST_ID=136
```

### DoD
For requester to complete:
